sigle/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["next/babel"],
-  "plugins": ["babel-plugin-macros", ["styled-components", { "ssr": true }]]
+  "plugins": ["babel-plugin-macros"]
```

Comment:
this will likely create some issues but using radix and this at the same time is not working 😀

Reply:
YOLO